index.html

```diff
@@ -0,0 +1,223 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+      <div class="container">
+        <div class="header">
```

Comment:
Well done implementing the requirements but there are some issues you need to fix to proceed.

The header should have a header tag instead of div tag
these above requirements are necessary to follow best practices, better semantics, and for a better user experience.